lib/heroku_rails_deflate/serve_zipped_assets.rb

```diff
@@ -42,7 +42,7 @@ def call(env)
             headers['Content-Type'] = Rack::Mime.mime_type(File.extname(path), 'text/plain')
 
             # Update cache-control to add directive telling Rack::Deflate to leave it alone.
-            cache_control = headers['Cache-Control'].try(:to_s).try(:downcase)
+            cache_control = headers['Cache-Control']&.to_s&.downcase
```

Comment:
I'm not a maintainer or anything but would love to see this merged as I'm working on migrating a codebase to rails 5 that uses it.

I'm pretty sure this use of `try` still works fine in rails 5 though, so this change doesn't seem necessary and it would break support for ruby < 2.3

Reply:
+1